packages/react-hooks/src/CredentialProvider.tsx

```diff
@@ -30,12 +30,22 @@ export const useCredentialById = (id: string): CredentialExchangeRecord | undefi
   return credentials.find((c: CredentialExchangeRecord) => c.id === id)
 }
 
-export const useCredentialByState = (state: CredentialState): CredentialExchangeRecord[] => {
+export const useCredentialByState = (
+  state: CredentialState | CredentialState[],
+  invertSearch = false
+): CredentialExchangeRecord[] => {
+  let states = typeof state === 'string' ? [state] : state
```

Comment:
```suggestion
  let states = [...state]
```

or

```suggestion
  const states = inverted ? Object.values(CredentialState).filter((v) => ![...state].includes(v)) : [...state]
```

But that might become a bit unreadable.

Reply:
Thanks for the feedback! the top part is good imho

Reply:
ah just a side note: `let states = [...state]` won't work. when state is a string it will create a character array of each char of the string instead an array with the one entry being the string.

Reply:
so not sure I can think of another/better way to get rid of the let then

Reply:
happy for suggestions

Reply:
Ah I was not aware spread operators work like that on strings, makes sense. 

Just leave it as-is then.

Reply:
I've added it to the other byId stuff. Lemme know if you have anymore feedback